src/coreclr/src/CMakeLists.txt

```diff
@@ -60,6 +60,12 @@ if(CLR_CMAKE_HOST_UNIX)
   add_subdirectory(nativeresources)
 endif(CLR_CMAKE_HOST_UNIX)
 
+# Workaround for errors on GCC
+if ((CMAKE_CXX_COMPILER_ID STREQUAL "GNU") AND (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 10.0))
+    add_compile_options(-Wno-error=stringop-overflow=)
```

Comment:
Is the `=` sign at the end intentional?

Reply:
I think it can be simplified to just ` add_compile_options(-Wno-stringop-overflow)`.

Reply:
> Is the `=` sign at the end intentional?

Yes.

> I think it can be simplified to just `add_compile_options(-Wno-stringop-overflow)`.

After switching to `add_compile_options(-Wno-stringop-overflow)`, the test still found errors:
```
  In file included from /home/heiher/git/dotnet/runtime/src/coreclr/src/ToolBox/superpmi/superpmi-shared/standardpch.h:68,
                   from /home/heiher/git/dotnet/runtime/src/coreclr/src/ToolBox/superpmi/superpmi-shared/methodcontextreader.cpp:11:
  In member function ‘void std::basic_string<T>::copy(const std::basic_string<T>&) [with T = char]’,
      inlined from ‘std::basic_string<T>& std::basic_string<T>::assign(const std::basic_string<T>&) [with T = char]’ at /home/heiher/git/dotnet/runtime/src/coreclr/src/inc/clr_std/string:80:13,
      inlined from ‘std::basic_string<T>::basic_string(const std::basic_string<T>&) [with T = char]’ at /home/heiher/git/dotnet/runtime/src/coreclr/src/inc/clr_std/string:44:15,
      inlined from ‘std::string to_lower(const string&)’ at /home/heiher/git/dotnet/runtime/src/coreclr/src/ToolBox/superpmi/superpmi-shared/methodcontextreader.cpp:32:23,
      inlined from ‘static std::string MethodContextReader::CheckForPairedFile(const string&, const string&, const string&)’ at /home/heiher/git/dotnet/runtime/src/coreclr/src/ToolBox/superpmi/superpmi-shared/methodcontextreader.cpp:49:87:
  /home/heiher/git/dotnet/runtime/src/coreclr/src/inc/clr_std/string:94:25: error: ‘void* __builtin_memcpy(void*, const void*, long unsigned int)’ accessing 18446744073709551615 bytes at offsets 0 and 0 overlaps 9223372036854775807 bytes at offset -9223372036854775808 [-Werror=restrict]
     94 |             m_string[i] = _Right.m_string[i];
        |             ~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
```

Reply:
`-Werror=restrict` is a different error?

Reply:
`-Werror=restrict` occurs when only switching from `add_compile_options(-Wno-error=stringop-overflow=)` to `add_compile_options(-Wno-stringop-overflow)`. I think the two errors are related, may be strings error  detection has changed.